lib/flight_scheduler/job.rb

```diff
@@ -126,7 +126,8 @@ def time_out?
     def start_time_out_task
       return if @time_out.nil?
       Async do |task|
-        Async.logger.info "Job '#{id}' will start timing out in '#{@time_out}'"
+        remaining_time = @time_out + @created_time - Process.clock_gettime(Process::CLOCK_MONOTONIC)
+        Async.logger.info "Job '#{id}' will start timing out in '#{remaining_time.to_i}'"
```

Comment:
We should add `seconds` to the end of this log message.

```
Job '945668ca-4695-4a30-87a0-b9efd00a42b8' will start timing out in '59'
```

Would be clearer if it told me `59` what.

Reply:
https://github.com/openflighthpc/flight-scheduler-daemon/pull/29/commits/991276724d5f5cf0fc299ec2192732c35e1fbc1f